catalogs/catalog-jdbc-doris/src/main/java/com/datastrato/gravitino/catalog/doris/operation/DorisTableOperations.java

```diff
@@ -0,0 +1,619 @@
+      try {
+        String sql =
+            "SELECT TABLE_COMMENT FROM information_schema.TABLES WHERE TABLE_SCHEMA = ? AND TABLE_NAME = ?";
+        String comment = "";
+        PreparedStatement preparedStatement = connection.prepareStatement(sql);
+        preparedStatement.setString(1, databaseName);
+        preparedStatement.setString(2, tableName);
+
+        ResultSet resultSet = preparedStatement.executeQuery();
+        while (resultSet.next()) {
+          comment += resultSet.getString("TABLE_COMMENT");
+        }
+
+        tableBuilder.withComment(comment);
+      } catch (SQLException e) {
+        throw exceptionMapper.toGravitinoException(e);
+      }
+    }
+  }
+
+  @Override
+  protected String generateRenameTableSql(String oldTableName, String newTableName) {
+    return String.format("ALTER TABLE `%s` RENAME `%s`", oldTableName, newTableName);
+  }
+
+  @Override
+  protected String generateDropTableSql(String tableName) {
+    return String.format("DROP TABLE `%s`", tableName);
+  }
+
+  @Override
+  protected String generatePurgeTableSql(String tableName) {
+    return String.format("TRUNCATE TABLE `%s`", tableName);
+  }
+
+  @Override
+  protected String generateAlterTableSql(
+      String databaseName, String tableName, TableChange... changes) {
+    // Not all operations require the original table information, so lazy loading is used here
+    JdbcTable lazyLoadTable = null;
+    TableChange.UpdateComment updateComment = null;
+    List<TableChange.SetProperty> setProperties = new ArrayList<>();
+    List<String> alterSql = new ArrayList<>();
+    for (int i = 0; i < changes.length; i++) {
+      TableChange change = changes[i];
+      if (change instanceof TableChange.UpdateComment) {
+        updateComment = (TableChange.UpdateComment) change;
+      } else if (change instanceof TableChange.SetProperty) {
+        // The set attribute needs to be added at the end.
+        setProperties.add(((TableChange.SetProperty) change));
+      } else if (change instanceof TableChange.RemoveProperty) {
+        // Doris only support set properties, remove property is not supported yet
+        throw new IllegalArgumentException("Remove property is not supported yet");
+      } else if (change instanceof TableChange.AddColumn) {
+        TableChange.AddColumn addColumn = (TableChange.AddColumn) change;
+        lazyLoadTable = getOrCreateTable(databaseName, tableName, lazyLoadTable);
+        alterSql.add(addColumnFieldDefinition(addColumn));
+      } else if (change instanceof TableChange.RenameColumn) {
+        throw new IllegalArgumentException("Rename column is not supported yet");
+      } else if (change instanceof TableChange.UpdateColumnType) {
+        lazyLoadTable = getOrCreateTable(databaseName, tableName, lazyLoadTable);
+        TableChange.UpdateColumnType updateColumnType = (TableChange.UpdateColumnType) change;
+        alterSql.add(updateColumnTypeFieldDefinition(updateColumnType, lazyLoadTable));
```

Comment:
Please review the backquote issues regarding column names, table names, etc in the whole PR carefully.